src/IECoreGL/Shader.cpp

```diff
@@ -665,7 +666,8 @@ class Shader::Setup::MemberData : public IECore::RefCounted
 		};
 
 		ConstShaderPtr shader;
-		vector<ValuePtr> values;
+		boost::container::flat_map<GLint, ValuePtr> vertexValues;
+		boost::container::flat_map<GLint, ValuePtr> uniformValues;
```

Comment:
I think uniform and vertex inputs share the same namespace in the GLSL source, so it would OK/logical to store them together in the one map.

Reply:
It seems they don't the same namespace.  I found some people on StackOverflow saying they don't share the same namespace, but without proper sourcing.  Just to confirm, I stuck them in the same map, and got piles of test failures, so that makes me feel better about test coverage.